src/script/main/app.ts

```diff
@@ -384,7 +389,7 @@ export class App {
         userRepository.addClientToUser(userId, newClient, true);
       });
 
-      const selfUser = await this.initiateSelfUser();
+      await this.initiateSelfUser(selfUser);
```

Comment:
We can only initialize the self user here, since we need the DB to have been created by the core